### PR TITLE
Mark `dart:ffi` as stable

### PR DESCRIPTION
**Pull request by dcharkes**
Should be merged when https://dart-review.googlesource.com/c/sdk/+/179765 is merged.

### CODE REVIEW

**Comment by sfshaza2** [comment]
@kwalrath, is dart:ffi stable? The link is rather confusing.

**Comment by kwalrath** [comment]
Not yet. We should merge this when we merge https://github.com/dart-lang/site-www/pull/2873.

File: src/docs/development/platform-integration/c-interop.md [hunk 1]
```diff
@@ -22,9 +22,10 @@ In this walkthrough, you'll create a C function
 that implements 32-bit addition and then
 exposes it through a Dart plugin named "native_add".
 
-{{ site.alert.note }}
-  The dart:ffi library is [in beta][ffi issue],
-  and breaking API changes might still happen.
+{{ site.alert.version-note }}
```
Comment (kwalrath) [inline src/docs/development/platform-integration/c-interop.md:25]: I just checked, and flutter.dev doesn't define version-note. @sfshaza2 would you like me to add a version note like on dart.dev, or would you prefer another solution?

**Comment by sfshaza2** [comment]
Where is this version note? I don't see it on dart.dev itself.

**Comment by kwalrath** [comment]
> Where is this version note? I don't see it on dart.dev itself.

You can see an example of a version note at https://dart.dev/tools/sdk.

**Comment by sfshaza2** [comment]
I looked at that note and I'm still not sure what you are suggestion, but I
just landed this PR which might satisfy what you are talking about?
https://github.com/flutter/website/pull/5205

File: src/docs/development/platform-integration/c-interop.md (hunk 1 above)
Comment (kwalrath) [reply src/docs/development/platform-integration/c-interop.md:25]: We talked out-of-band and agreed that I'll add the version-note to _config.yml. (I'll do it in a separate PR.)

**Comment by sfshaza2** [comment]
Thx!

File: src/docs/development/platform-integration/c-interop.md (hunk 1 above)
Comment (kwalrath) [reply src/docs/development/platform-integration/c-interop.md:25]: PR: #5216

Comment (sfshaza2) [reply src/docs/development/platform-integration/c-interop.md:25]: Let me know when you are ready for me to review.

Comment (kwalrath) [reply src/docs/development/platform-integration/c-interop.md:25]: Ready now. Here's a screenshot:

image

**Review by kwalrath — APPROVED** [review]

**Pull request merged**